types/worktreesave.go

```diff
@@ -0,0 +1,66 @@
+package types
```

Comment:
I created this just for a foreseeable scenario where `git reset` from go-git doesn't really work and we may destroy our customer's work when trying to remote build.